src/lookup_key.rs

```diff
@@ -334,7 +334,7 @@ impl PathItem {
 
 /// wrapper around `getattr` that returns `Ok(None)` for attribute errors, but returns other errors
 /// We dont check `try_from_attributes` because that check was performed on the top level object before we got here
-fn py_get_attrs<'a, 'b>(obj: &'a PyAny, attr_name: &'b Py<PyString>) -> PyResult<Option<&'a PyAny>> {
+fn py_get_attrs<'a>(obj: &'a PyAny, attr_name: &Py<PyString>) -> PyResult<Option<&'a PyAny>> {
```

Comment:
cargo lint raised an error, saying that lifetime `'b` could be elided.

Reply:
:+1: